WORKSPACE

```diff
@@ -1,52 +1,73 @@
-workspace(name = "angular_devkit")
+workspace(name = "angular_cli")
```

Comment:
note, this change has downstream implications for bazel usage (in angular/angular we load this workspace to get build optimizer)

Reply:
Hadn't considered that. Perhaps it's better to leave it as is. WDYT?